ember/ember.d.ts

```diff
@@ -1765,9 +1788,43 @@ declare module Ember {
         toArray(): any[];
     }
     module RSVP {
+        interface IPromiseFunction {

```

Comment:
The TypeScript [handbook](http://www.typescriptlang.org/Handbook#interfaces), [language specification](http://www.typescriptlang.org/Content/TypeScript%20Language%20Specification.pdf) and [wiki](https://github.com/Microsoft/TypeScript/wiki/Interfaces) all name their interfaces without an `I` at the beginning. I read somewhere – though I can't find it now – that they suggest not prefixing interfaces with the letter `I` and that's why you don't see any interfaces in this Ember definition file named that way. If nothing else, please be consistent with the patterns already in place.